### DISCUSSION TITLE
Cannot find any options for changing display name for fields

### DISCUSSION THREAD

**Discussion by lindesvard** [strapi/strapi#6595] [Legacy Topics]
Looked at the documentation for the attributes and I just the validation options. Have a field I want to rename in admin ui (not in the api response).

Added this as a discussion and not a feature request because I don't know if it exists or not. 

If it not exists some more settings would be useful. 

`displayName`/`name`: Changing name in admin ui
`description`: A little explanation what this field does

New to strapi and love it so far!

**Comment by derrickmehaffy** [comment]
There is no way to set it via files currently (I have a PR to answer this question that will be added to the FAQ section of the docs, they just haven't been redeployed yet) https://github.com/strapi/strapi/blob/b17f4739023e57f01b41207ce05de4925db7146b/docs/v3.x/getting-started/troubleshooting.md#can-i-store-my-content-manager-layout-configurations-in-the-model-settings

But other than that you would set them via the content-manager and you can use the cli tools to dump these and restore them.

https://strapi.io/documentation/v3.x/cli/CLI.html#strapi-configuration-dump-config-dump
https://strapi.io/documentation/v3.x/cli/CLI.html#strapi-configuration-restore-config-restore
